src/Microsoft.AspNet.Identity/RoleManager.cs

```diff
@@ -345,8 +364,12 @@ private IRoleClaimStore<TRole> GetClaimStore()
             {
                 throw new ArgumentNullException("role");
             }
-            await claimStore.AddClaimAsync(role, claim, CancellationToken);
-            return await LogResultAsync(await UpdateRoleAsync(role), role);
+
+            using (Logger.BeginScope(await GetScopeMessageforRoleAsync(role)))
```

Comment:
Can this hang of IdentityLogger class ? Then that class can be one place where we can edit and would reflect in all there Manager classes instead of editing 3 private helper methods